File: src/Iniciante/1013.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int zega(int a, int b, int c)
{
    int m, n;
    m = ((a + b + abs(a - b)) / 2);
    n = ((m + c + abs(m - c)) / 2);
    return (n);
}

int main()
{

    int v1, v2, v3, n;
    scanf("%d %d %d", &v1, &v2, &v3);
    n = zega(v1, v2, v3);
    printf("%d eh o maior\n", n);

    return 0;
}

File: src/Iniciante/1013.java
package Iniciante;

import java.io.IOException;
import java.util.Scanner;


public class Main {
    public static void main(String[] args) throws IOException {
      
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();
        int c = input.nextInt();
        
        int mab= (a+b+Math.abs(a-b))/2;
        int mabc = (mab + c + Math.abs(mab - c))/2;

        
        System.out.println(mabc+" eh o maior");    
       

        }
    
}
